### PR TITLE
BinaryToDecimal

### PR DESCRIPTION
**Pull request by shalini-tiwari**
BinaryToDecimal

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+num = int(input("enter any binary number"))
+sum = 0
+i = 0
+while num!=0:
+    rem = num % 10
+    sum = sum + rem * pow(2,i)
+    num = int(num/1o)
```
Comment (DeepNinja07x) [inline main.py]:
```suggestion
    num = int(num/10)
```
Just do the correction by removing __'o'__ with 0.

File: main.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+num = int(input("enter any binary number"))
+sum = 0
+i = 0
+while num!=0:
+    rem = num % 10
+    sum = sum + rem * pow(2,i)
+    num = int(num/1o)
+    i = i+1
+    print("Decimal number :",sum)
```
Comment (DeepNinja07x) [inline main.py]: And make the ``` print(Decimal number:", sum)``` outside the while loop

**Review by DeepNinja07x — CHANGES_REQUESTED** [review]
Just do the following changes @shalini-kumari2000. The rest of the code is fine.

File: main.py (hunk 1 above)
Comment (shalini-tiwari) [reply main.py]: yaa done

**Review by subhayu99 — APPROVED** [review]

**Comment by subhayu99** [comment]
@shalini-kumari2000 Try to name the PR according to the changes you made. Rest is fine✌🏼

**Review by subhayu99 — APPROVED** [review]

**Review by DeepNinja07x — APPROVED** [review]

**Pull request merged**